header-validator/src/validate-os.ts

```diff
@@ -1,16 +1,18 @@
-import { Issue, ValidationResult } from './issue'
-const { parseList } = require('structured-headers')
+import { Context, ValidationResult } from './context'
+import { InnerList, Item, parseList } from 'structured-headers'
 
-function validateURL(item: any): string|undefined {
-  if (typeof item !== 'string') {
-    return 'must be a string'
+function validateURL(ctx: Context, member: InnerList|Item): void {
+  if (typeof member[0] !== 'string') {
```

Comment:
Might be useful to spread the values in the member tuple to document what they represent.